secretmanager/snippets/regional_samples/delete_regional_secret.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for deleting an existing regional
+secret.
+"""
+
+import argparse
+
+
+# [START secretmanager_v1_delete_regional_secret]
+def delete_regional_secret(project_id: str, location_id: str, secret_id: str) -> None:
+    """
+    Delete the secret with the given name and all of its versions.
```

Comment:
be consistent: `create_regional_secret` was fully documented with Args section and an example, so include it in all of the samples being added or remove it in all.

Reply:
I will remove them for now because of the time constraints. 
I will create a bug and will add them if need arise.